cips/cip-14.md

```diff
@@ -215,17 +215,16 @@ This proposal is backwards-incompatible because it is state-machine breaking. Th
 
 The following test cases are available in the [ibc-go e2e repository.](https://github.com/cosmos/ibc-go/tree/main/e2e/tests/interchain_accounts)
 
-- Registration of an interchain account
-- Transfer funds from interchain account to a different account on the same chain using an unordered channel
-- Transfer funds from interchain account to a different account on the same chain using an ordered channel
-- A failed transfer of funds from interchain account to a different account on the same chain due to insufficient funds in the interchain account balance
-- Transfer funds from interchain account to a different account on the same chain after an ordered channel closes and a new channel is reopened to connect to the existing interchain account
-- A transfer of funds from an interchain account to a different account on the same chain using an x/gov sdk module based controller (on the controlling chain)
-- A transfer of funds from an interchain account to a different account on the same chain using a x/group sdk module based controller (on the controlling chain)
-- [OPTIONAL] A transfer of funds from an interchain account to a different account on the same chain using an incentivised IBC packet
-- Registration of an interchain account, and transfer of funds from an interchain account to a different account on the same chain through the localhost client, and after reopening a channel
-- Query if host functionality is enabled
-- Transfer funds from interchain account to a different account after upgrading the channel from ordered to unordered
+- Registration of an interchain account - [test link](https://github.com/cosmos/ibc-go/blob/main/e2e/tests/interchain_accounts/base_test.go#L46)
+- Transfer funds from interchain account to a different account on the same chain using an unordered channel - [test link](https://github.com/cosmos/ibc-go/blob/main/e2e/tests/interchain_accounts/base_test.go#L56)
```

Comment:
> using an unordered channel

This seems like it has a prerequisite on ibc-go [v8.1.0](https://github.com/cosmos/ibc-go/releases/tag/v8.1.0) because v8.1.0 release notes include:

> This release adds support for opening new ICA channels using unordered ordering

```suggestion
- [OPTIONAL] Transfer funds from interchain account to a different account on the same chain using an unordered channel - [test link](https://github.com/cosmos/ibc-go/blob/main/e2e/tests/interchain_accounts/base_test.go#L56). Note: requires ibc-go >= v8.1.0.
```